src/TimeWarpEventSet.cpp

```diff
@@ -19,9 +20,9 @@ void TimeWarpEventSet::initialize (const std::vector<std::vector<LogicalProcess*
     /* Create the input and processed queues and their locks.
        Also create the input queue-scheduler map and scheduled event pointer. */
     input_queue_lock_ = make_unique<std::mutex []>(num_of_lps);
-#ifdef SCHEDULE_QUEUE_SPINLOCKS
+#if defined (SCHEDULE_QUEUE_SPINLOCKS)
```

Comment:
If someone accidentally defines both ONE_THREAD_PER_LTSF and SCHEDULE_QUEUE_SPINLOCKS, schedule_queue_lock_ would be initialized with TicketLocks(). So ideally the structure should be:
#if !defined(ONE_THREAD_PER_LTSF)
#if defined(SCHEDULE_QUEUE_SPINLOCKS)
<create the ticket locks here>
#else
<create the mutexes here>
#endif
#endif